### PR TITLE
Remove secrets

### PR DESCRIPTION
**Pull request by jmccann**
As being discussed in #44 I am removing secrets key and related code.

I also added searching for any environment variables with `TF_VAR_` prefix and copy them to a lowercase version.  e.g. `TF_VAR_SOMETHING` -> `TF_VAR_something`.  Both would have the same value.  This is due to Drone taking anything in `environment` and creating with with uppercase.

Appreciate feedback from @josmo @msuterski and @edwinavalos

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -103,9 +102,14 @@ func installCaCert(cacert string) *exec.Cmd {
...
-func exportSecrets(secrets map[string]string) {
-	for k, v := range secrets {
-		os.Setenv(fmt.Sprintf("%s", k), fmt.Sprintf("%s", os.Getenv(v)))
+func CopyTfEnv() {
+	tfVar := regexp.MustCompile(`^TF_VAR_.*$`)
+	for _, e := range os.Environ() {
+		pair := strings.Split(e, "=")
+		if tfVar.MatchString(pair[0]) {
+			name := strings.Split(pair[0], "TF_VAR_")
+			os.Setenv(fmt.Sprintf("TF_VAR_%s", strings.ToLower(name[1])), pair[1])
```
Comment (msuterski) [inline plugin.go]: It's a bit of a slippery slope to lowercase everything. A name of a variable in TF does not need to be all lowercase. I could have something like `TF_VAR_myListOfCIDRs` which won't work/get fixed with this change. 

I think the problem is on the drone side, where it's uppercasing everything, maybe unnecessarily, if there are use cases what would not want it to happen. I feel like fixing it in the plugin is not a right place to do so.

File: plugin_test.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"os"
+	"testing"
+	. "github.com/franela/goblin"
+)
+
+func TestPlugin(t *testing.T) {
```
Comment (msuterski) [inline plugin_test.go]: 👍

File: plugin.go (hunk 1 above)
Comment (jmccann) [inline plugin.go]: I agree.  And I meant to bring it up to @bradrydzewski (hi brad) ... this was going to be more of a "band-aid" in the meantime.

**Comment by jmccann** [comment]
@msuterski Thanks for taking the time to review and comment!

File: plugin.go (hunk 1 above)
Comment (jmccann) [inline plugin.go]: FYI, found https://github.com/cncd/pipeline/search?utf8=✓&q=ToUpper&type=

**Comment by jmccann** [comment]
Created an RC build with this code and it seemed to work

This example is setting env with secrets for open stack provider (`os_username ` and `os_password`) as well as terraform variable from secrets via env with `tf_var_some_secret`

```yaml
  terraform_plan:
    when:
      branch:
        exclude: master
    secrets:
      - source: some_secret
        target: tf_var_some_secret
      - source: os_username
        target: os_username
      - source: os_password
        target: os_password
    image: jmccann/drone-terraform:rc
    pull: true
    plan: true
    init_options:
      backend-config:
        - "path=some/path"
    vars:
      vara: somevalue
```

**Pull request merged**